sympy/vector/integrals.py

```diff
@@ -150,10 +153,33 @@ def vector_integrate(field, *region):
     >>> vector_integrate(C.x*C.i, region)
     12
 
+    Integrals over special regions can also be calculated using geometry module.
+    >>> from sympy.geometry import Point, Circle, Triangle
+    >>> c = Circle(Point(0, 2), 5)
+    >>> vector_integrate(C.x**2 + C.y**2, c)
+    290*pi
+    >>> triangle = Triangle(Point(-2, 3), Point(2, 3), Point(0, 5))
+    >>> vector_integrate(3*C.x**2*C.y*C.i + C.j, triangle)
+    -8
+
+    >>> vector_integrate(12*C.y**3, (C.y, 1, 3))
+    240
     >>> vector_integrate(C.x**2*C.z, C.x)
     C.x**3*C.z/3
+
     """
     if len(region) == 1:
         if isinstance(region[0], ParametricRegion):
             return ParametricIntegral(field, region[0])
+
+        if isinstance(region[0], GeometryEntity):
+            if isinstance(region[0], Polygon):
+                result = 0
+                for s in region[0].sides:
+                    result += vector_integrate(field, s)
+                return result
```

Comment:
I see a problem here in the case the GeometryEntity is not convex, you need to break it into more parametric regions. Maybe `parametric_region( )` should instead return a list of parametric regions? So you can also avoid the `if` clause as defined above.